contracts/UpkeepRegistrationRequests.sol

```diff
@@ -95,7 +99,7 @@ contract UpkeepRegistrationRequests is Owned {
       external
       onlyLINK()
     {
-        bytes32 hash = keccak256(msg.data);
+        bytes32 hash = keccak256(abi.encode(upkeepContract, gasLimit, adminAddress, checkData, amount));
```

Comment:
```suggestion
        bytes32 hash = keccak256(abi.encode(upkeepContract, gasLimit, adminAddress, checkData));
```
I think you should take amount out of here, because otherwise you can only increase a deposit by the same amount you originally deposited.

Reply:
whoops, good catch